templates/default/nodewatcher.cfg.erb

```diff
@@ -1,4 +1,5 @@
 [nodewatcher]
 region = <%= node['cfncluster']['cfn_region'] %>
 scheduler = <%= node['cfncluster']['cfn_scheduler'] %>
-proxy = <%= node['cfncluster']['cfn_proxy'] %>
\ No newline at end of file
+proxy = <%= node['cfncluster']['cfn_proxy'] %>
+idle_time = <%= node['cfncluster']['cfn_scale_down_idle_time'] %>
```

Comment:
should we call this in a more complete way, `scaledown_idletime` or anyway something more than `idle_time` ?

Reply:
Yes, scaledown_idletime sounds good.